Python/codegen.c

```diff
@@ -686,12 +689,13 @@ codegen_setup_annotations_scope(compiler *c, location loc,
 }
 
 static int
-codegen_leave_annotations_scope(compiler *c, location loc,
-                                Py_ssize_t annotations_len)
+codegen_leave_annotations_scope(compiler *c, location loc)
 {
-    ADDOP_I(c, loc, BUILD_MAP, annotations_len);
     ADDOP_IN_SCOPE(c, loc, RETURN_VALUE);
     PyCodeObject *co = _PyCompile_OptimizeAndAssemble(c, 1);
```

Comment:
I'm on mobile but aren't we leaking this one in case of an error? We check for nullability of this later (which seems wrong as we access one of it's member) and we decref it only at the end and not before returning ERROR sometimes. Like if size is incorrect we don't decref this one.

Reply:
I think you're right, that's a pre-existing bug but I'll just fix it here as the code is pretty different on 3.13 anyway.